sdk/rust-examples/huffman-encoding/README.md

```diff
@@ -0,0 +1,41 @@
+# A simple huffman encoding implementation with Veracruz freestanding execution engine
+
+
+This is a simple program that lets you implement the "Huffman Encoding Algorithm" which is a Lossless Compression Algorithm used to compresss Data. It was developed by David A. Huffman.  
+This directory is a simple program that takes a file called `huffman_example_input.txt` and compresses its data using the Huffman Encoding Algorithm, and then writes teh output into `encoded_output.txt`
+
+# Steps
+
+1. Go to `sdk/freestanding-execution-engine` directory.

```

Comment:
Is it really necessary to go to `freestanding-execution-engine` to call the FEE and pass the program to it?
Can't they be under different directories?

Reply:
tbh I know little on this topic, and will get back to it after resolving the other conversations. Really sorry for this, but it might take me some time!!